debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseSchema.java

```diff
@@ -29,6 +29,7 @@
  */
 public abstract class RelationalDatabaseSchema implements DatabaseSchema<TableId> {
     private final static Logger LOG = LoggerFactory.getLogger(RelationalDatabaseSchema.class);
+    public static final String NO_MONITORED_TABLES_WARNING = "After applying include/exclude list filters there are no tables to monitor, please check your configuration";
```

Comment:
Was moved here to avoid duplicate/copied strings in different connectors and their tests.

Reply:
As commented elsewhere, needs to go to the super-class probably.

Reply:
will do